helm/configs/1_event_attributes.conf

```diff
@@ -12,6 +12,7 @@ commands = [
               fqn: Span.id,
               key: id,
               value_kind: TYPE_STRING,
+              groupable: true,
```

Comment:
I thought `true` is the default value for all string attributes.

Reply:
Because of the config bootstrapper set up, all attribute definitions get pumped through proto serde so appear with the proto defaults. For booleans, that'd be false - we have to set them explicitly.